docs/notebooks/thomson.ipynb

```diff
@@ -0,0 +1,181 @@
+    "              'fract':np.array([1]), 'ion_z':np.array([1]), 'ion_mu':np.array([1]),\n",
+    "              'fluid_vel':np.array([0, 0, 0])*u.km/u.s, 'ion_vel':np.array([[0, 0, 0]])*u.km/u.s}\n",
+    "\n",
+    "# Identical to the collective example above, except now the electron fluid has a substantial drift velocity parallel to the\n",
+    "# probe laser and the ions have a drift (relative to the electrons) at an angle.\n",
+    "drifts = {'name':'Drift Velocities', 'ne':5e17*u.cm**-3, 'Te':10*u.eV, 'Ti':np.array([10])*u.eV,\n",
+    "          'fract':np.array([1]), 'ion_z':np.array([1]), 'ion_mu':np.array([1]),\n",
+    "          'fluid_vel':np.array([700, 0, 0])*u.km/u.s, 'ion_vel':np.array([[-600, -100, 0]])*u.km/u.s}\n",
+    "\n",
+    "# Identical to the collective example, except now the plasma consists 25% He+1 and 75% C+5\n",
+    "two_species = {'name':'Two Ion Species', 'ne':5e17*u.cm**-3, 'Te':10*u.eV, 'Ti':np.array([10, 50])*u.eV,\n",
+    "               'fract':np.array([.25, .75]), 'ion_z':np.array([1,5]), 'ion_mu':np.array([4,12]),\n",
+    "               'fluid_vel':np.array([0, 0, 0])*u.km/u.s, 'ion_vel':np.array([[0, 0, 0],[0, 0, 0]])*u.km/u.s}\n",
+    "\n",
+    "examples = [non_collective, weakly_collective, collective, drifts, two_species]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define some parameters that we won't change between the examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The probe wavelength can in theory be anything, but in practice integer frequency multiples of the Nd:YAG wavelength\n",
+    "# 1064 nm are used (532 corresponds to a frequency-doubled probe beam from such a laser).\n",
+    "probe_wavelength = 532*u.nm\n",
+    "\n",
+    "# Array of wavelengths over which to calcualte the spectral distribution\n",
+    "wavelengths = np.arange(probe_wavelength.value-60, probe_wavelength.value+60, 0.01)*u.nm\n",
+    "\n",
+    "# The scattering geometry is defined by unit vectors for the orientation of the probe laser beam (probe_n) and\n",
+    "# the path from the scattering volume (where the measurement is made) to the detector (scatter_n).\n",
+    "# These can be setup for any experimental geometry.\n",
+    "probe_vec = np.array([1, 0, 0])\n",
+    "scattering_angle = np.deg2rad(63)\n",
+    "scatter_vec = np.array([np.cos(scattering_angle), np.sin(scattering_angle), 0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For each example, plot the the spectral distribution function over a large range to show the broad electron scattering feature (top row) and a narrow range around the probe wavelength to show the ion scattering feature (bottom row)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
```

Comment:
In order to get a thumbnail for the `nbsphinx` gallery, you need add metadata to a specific cell to tell `nbsphinx` where it should extract the image from.  This cell is probably the best candidate.  The metadata would look something like...

```suggestion
   "metadata": {
    "nbsphinx-thumbnail": {
     "tooltip": "I am a tool tip.",
     "output-index": 3
       }
   },
```
https://nbsphinx.readthedocs.io/en/0.7.0/gallery/cell-metadata.html
https://nbsphinx.readthedocs.io/en/0.7.0/gallery/multiple-outputs.html

Reply:
And we should really document that as well, shouldn't we...